### ISSUE TITLE
Not compile

### ISSUE THREAD

**Issue by jbarriosmuriel** [AEmelyanov757/a9g_traccar#1]
I can't compile the code.

<img width="942" alt="Captura" src="https://user-images.githubusercontent.com/13985328/75037071-a148bf80-54b3-11ea-89bd-ef5fdc2b0028.PNG">

**Comment by AEmelyanov757** [comment]
Thanks for the message. 
Yes, this is my typo when transferring the project to github. It is necessary to replace the code "LOCAL_NAME := demo/gps_tracker" with "LOCAL_NAME := ag_tracker"in the Makefile on line 6. 
On March 1, 2020, I plan to update the code with a new version, so don't rush to fix the bug.

**Comment by jbarriosmuriel** [comment]
Ok, thank you!.  Can you let me know when you launch it?. Writing here for example.

Thanks for your attention.
Regards

**Comment by jbarriosmuriel** [comment]
The port for a9p in traccar is 5005 or 5055?. En another tutorial https://xcode.vn/sim-a9g-bai-6-gps-tracking-voi-traccar-server/  recommend the 5055, but in your code is 5005. Have you successfully connected traccar and send GPS coordinates ?. I am able to connect but it always show the point in the 90 latitude and 0 longitude. However, However, some time ago I was able to show correct coordinates, but I don't remember which port I chose.

Thanks for your response
